flake8.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length= 120
 exclude=
-ignore=E252,W504,W503
+ignore=W503
```

Comment:
Looks like we can remove this as well? When I run flake8 on `main` I only see `E252` errors which you have resolved all here.

Reply:
W503 and W504 are mutually exclusive. One says "thou must put the + on the first line" and the other "thou must put the + on the second line". PEP8 changed its mind about that at some point and W504 is what's canonical now, which means we need to ignore W503.

one or the other :sweat_smile:

Reply:
https://www.flake8rules.com/rules/W503.html and https://www.flake8rules.com/rules/W504.html

This was changed in 2016 yet both warnings remain in flake8, at least in the version I'm running ... go figure

Reply:
😭